src/averageAges.js

```diff
@@ -20,6 +20,20 @@ function calculateMenAverageAge(people, century) {
   // avoid using loop and forEach
   // replace `if ()` statement with &&, || or ?:
   // without nesting
+  const filteredMen = people.filter(person => person.sex === 'm'
+    && (century
+      ? Math.ceil(person.died / 100) === century
+      : true));
+
+  const menAges = filteredMen.map(man => man.died - man.born);
+
+  return getAverage(menAges, filteredMen.length);
+}
+
+function getAverage(ages, length) {
```

Comment:
do we need `length` here?

Reply:
not length, but quantity? function getAverage(ages, quantity) ?